public/characterCreator.html

```diff
@@ -1 +1,96 @@
-an interactive character creator 
\ No newline at end of file
+an interactive character creator
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
```

Comment:
if any type of ie => just give a message saying to upgrade to a better browser

Reply:
Yea I need to clean it up, I basically left in all of the boilerplate stuff

Reply:
Using this now

`<!--[if IE]>
            <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="#">upgrade your browser</a> to improve your experience.</p>
        <![endif]-->`